server/image_fs_info.go

```diff
@@ -38,7 +56,18 @@ func (s *Server) ImageFsInfo(context.Context, *types.ImageFsInfoRequest) (*types
 		return nil, err
 	}
 
-	return &types.ImageFsInfoResponse{
-		ImageFilesystems: []*types.FilesystemUsage{fsUsage},
+	return fsUsage, nil
+}
+
+func getUsage(containerPath string) (*types.FilesystemUsage, error) {
+	bytes, inodes, err := crioStorage.GetDiskUsageStats(containerPath)
+	if err != nil {
+		return nil, err
```

Comment:
let's wrap the errors:
```suggestion
		return nil, fmt.Errorf("get disk usage for path %s: %w", containerPath, err)
```